### PR TITLE
2020 06 19 add wifi settings

### PR DESCRIPTION
**Pull request by xkevin190**
- added global IP verification and valid AP to start udpServer
- internal connection view to wifi
- app lock if you are not connected to a valid network
- Native module for internal Wi-Fi connections and static IP configuration

this  PR is missing some details for finished, but you can already see the code structure.

### CODE REVIEW

**Comment by luisan00** [comment]
seems 10 tests are failing @xkevin190

File: android/app/src/main/java/DeviceInfo/DeviceInfoModule.java [hunk 1]
```diff
@@ -47,7 +49,7 @@ public String getName() {
     public Map<String, Object> getConstants() {
         final Map<String, Object> constants = new HashMap<>();
         constants.put("VersionInfo", getVersionInfo());
-        constants.put("globalIpv6", getIpv6());
+//        constants.put("globalIpv6", getIpv6());
```
Comment (luisan00) [inline android/app/src/main/java/DeviceInfo/DeviceInfoModule.java:52]: delete unused code
```suggestion
```

File: android/app/src/main/java/WiFiModule/WifiModule.java [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+//        int netId = wifi.addNetwork(wc);
+//        wifi.disconnect();
+//       if( wifi.enableNetwork(netId, true)){
+//           wifi.reconnect();
+//           IntentFilter intentFilter = new IntentFilter();
+//           intentFilter.addAction(ConnectivityManager.CONNECTIVITY_ACTION);
+//           intentFilter.addAction(WifiManager.SUPPLICANT_STATE_CHANGED_ACTION);
+//
+//           context.registerReceiver(connectedToLocalWifiReceiver, intentFilter);
+//       }
```
Comment (luisan00) [inline android/app/src/main/java/WiFiModule/WifiModule.java:130]: delete unused

```suggestion
```

**Comment by xkevin190** [comment]
finished functionality ready for review.

**Pull request merged**